.phpsa.yml

```diff
@@ -95,6 +99,9 @@ phpsa:
         # Checks for use of deprecated functions and gives alternatives if available.
         deprecated_functions:
             enabled:              true
+            check_5_3:            true
+            check_5_5:            true
+            check_7_1:            true
```

Comment:
Shouldn't we use the `language_level` setting to determine the checks that need to be done?